canonicalwebteam/blog/django/views.py

```diff
@@ -179,10 +179,10 @@ def author(request, username):
 def archives(request, template_path="blog/archives.html"):
     try:
         page = request.GET.get("page", default="1")
-        category = request.GET.get("category", default="")
         group = request.GET.get("group", default="")
         month = request.GET.get("month", default="")
         year = request.GET.get("year", default="")
+        category_param = request.GET.get("category", default="")
```

Comment:
This is now a bit confusing, as multiple Categories can be passed. 
We should probably rename all these params to their plural forms.

I will take note of that and make a major change to this later on, as this will require rewriting in all implementations.